src/normalized.jl

```diff
@@ -198,7 +198,7 @@ copy(L::Ldiv{Lay,<:WeightedBasisLayout{<:AbstractNormalizedOPLayout}}) where Lay
 copy(L::Ldiv{<:WeightedBasisLayout{<:AbstractNormalizedOPLayout},<:WeightedBasisLayout{<:AbstractNormalizedOPLayout}}) = copy(Ldiv{ApplyLayout{typeof(*)},ApplyLayout{typeof(*)}}(L.A,L.B))
 copy(L::Ldiv{<:WeightedOPLayout{<:AbstractNormalizedOPLayout},<:WeightedBasisLayout{<:AbstractNormalizedOPLayout}}) = copy(Ldiv{ApplyLayout{typeof(*)},ApplyLayout{typeof(*)}}(L.A,L.B))
 copy(L::Ldiv{<:WeightedBasisLayout{<:AbstractNormalizedOPLayout},<:WeightedOPLayout{<:AbstractNormalizedOPLayout}}) = copy(Ldiv{ApplyLayout{typeof(*)},ApplyLayout{typeof(*)}}(L.A,L.B))
-
+copy(L::Ldiv{L1,L2}) where {L1<:AbstractNormalizedOPLayout, L2<:WeightedOPLayout{<:AbstractNormalizedOPLayout}} = copy(Ldiv{ApplyLayout{typeof(*)},ApplyLayout{typeof(*)}}(L.A, L.B))
```

Comment:
To make more consistent with other calls:
```suggestion
copy(L::Ldiv{<:AbstractNormalizedOPLayout, <:WeightedOPLayout{<:AbstractNormalizedOPLayout}}) = copy(Ldiv{ApplyLayout{typeof(*)},ApplyLayout{typeof(*)}}(L.A, L.B))
```